rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -73,6 +76,18 @@ def generate_py(generator_arguments_file, typesupport_impls):
         idl_file = parse_idl_file(locator)
         idl_content.elements += idl_file.content.elements
 
+    # NOTE(sam): remove when a language specific name mangling is implemented
+    for message in idl_content.get_elements_of_type(Message):
+        for member in message.structure.members:
+            msg_name = message.structure.namespaced_type.name
+            if (keyword.iskeyword(member.name)):
+                print(('Member name "{}" in the message "{}" is a '
+                       'reserved keyword in Python and is not supported '
+                       'at the moment. Please use a different name.')
+                      .format(member.name, msg_name), file=sys.stderr)
```

Comment:
Please only use single indention steps and always a multiple of four spaces. Also one of the parenthesis pairs are redundant. Something like this instead:

```python
if (keyword.iskeyword(member.name)):
  print(
    'Member name "{}" in the message "{}" is a '
    'reserved keyword in Python and is not supported '
    'at the moment. Please use a different name.'
    .format(member.name, msg_name),
    file=sys.stderr)
```

Or if you want to keep the parenthesis make them do something specific like:

```python
if (keyword.iskeyword(member.name)):
  print(
    (
      'Member name "{}" in the message "{}" is a '
      'reserved keyword in Python and is not supported '
      'at the moment. Please use a different name.'
    ).format(member.name, msg_name),
    file=sys.stderr)
```

I prefer the first case though. The important thing though is to avoid this:

```
                print(('Member name "{}" in the message "{}" is a '
                       'reserved keyword in Python and is not supported '
              # ^-----^ this is more than 4 spaces
```

Reply:
Also, please use single quotes `'` to quote things being printed, and double quotes `"` to avoid needing escape characters. So `'Member name "{}" in the message "{}" is a '` -> `"Member name '{}' in the message '{}' is a "`.